cmd/minikube/cmd/config/get.go

```diff
@@ -38,9 +38,11 @@ var configGetCmd = &cobra.Command{
 		val, err := config.Get(args[0])
 		if err != nil {
 			fmt.Fprintln(os.Stdout, err)
+			os.Exit(1)
```

Comment:
This will work, but it might be worth looking into running something like https://github.com/spf13/cobra/blob/master/command.go#L96-L97
instead of `Run` as we're calling in our commands.

That should propagate up to https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/root.go#L104-L108